engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/BinaryOpFloatTest.java

```diff
@@ -4,7 +4,6 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-import com.google.common.collect.Streams;
```

Comment:
Why not using `com.google.common.collect.Streams` in `runtime-integration-tests`? I would understand removing that dependency from `runtim` itself, but not from tests. Besides, this only removes one or two usages and not the whole library.

Reply:
The library had been available by an accidental transitive dependency removed by `.dependsOn(interpreter-dsl % "provided")`. As the author of the original test, I swear, I would have never used a 3rd party library to simplify three line `Stream.concat` into a one-liner if I noticed it is a 3rd party library.

It is true, we don't need [lightweightness in tests](http://wiki.apidesign.org/wiki/HeavyWeight), but in this case I happily fix my previous mistake, which is [now](https://github.com/enso-org/enso/pull/9868/commits/751416d3351cf4dcd03b7e8bf1f2f1fe98306f50) apparent.